### PR TITLE
Updated manifests and dockerfile to use httpd instead of nginx

### PR DESCRIPTION
**Pull request by ciiay**
This PR is for [GITOPS-1323](https://issues.redhat.com/browse/GITOPS-1323).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,14 @@ WORKDIR /usr/src/app
 RUN yarn config set network-timeout 600000 -g
 RUN yarn install && yarn build
 
-FROM --platform=linux/amd64 docker.io/library/nginx:stable
+FROM --platform=linux/amd64 registry.redhat.io/rhel8/httpd-24
```
Comment (iam-veeramalla) [inline Dockerfile:8]: Do you want to specify the version ?

Something like `registry.redhat.io/rhel8/httpd-24:1-226`
or 
`registry.redhat.io/rhel8/httpd-24:1-230` ?

It depends on how confident we are that the new version of httpd does not cause any breaking changes for us.

File: Dockerfile [hunk 2]
```diff
@@ -5,7 +5,14 @@ WORKDIR /usr/src/app
...
 
-RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx
-COPY --from=build /usr/src/app/dist /usr/share/nginx/html
+USER 0
```
Comment (iam-veeramalla) [inline Dockerfile:10]: we can change this to `USER root`

**Review by iam-veeramalla — COMMENTED** [review]
PTAL

File: Dockerfile (hunk 1 above)
Comment (ciiay) [reply Dockerfile:8]: This is copied from kam, not sure if new version would cause any breaking changes for us.

Comment (iam-veeramalla) [reply Dockerfile:8]: Okay . Let's use it in that case.

**Review by iam-veeramalla — APPROVED** [review]
lgtm, Thanks @ciiay

**Pull request merged**